README.md

```diff
@@ -1,75 +1,87 @@
-      3.	AKSSPNClientIdRegion1 – client id for the region
-      4.	AKSSPNClientSecretRegion1 – client secret for the region
-      5.	TenantId – tenant id
-      6.	CSVFileNames – list of supported file names for execution like “users.csv,ids.csv”
-      
-      ![image](https://user-images.githubusercontent.com/81369583/114205527-0097b680-9978-11eb-90a4-45bd8c0a7326.png)
-5.	Set the mentioned pipeline variables as shown:
-   ![image](https://user-images.githubusercontent.com/81369583/114205558-08575b00-9978-11eb-8b1c-999b00f8e924.png)
-
-6.	Set the Variable group linked from Key vault.     
-
-7.	The results of the execution is published as artifact and it can be downloaded. The index.html file holds the report of the run.
+
+1. create the test suite with the help of how to setup JMeter test plan(https://jmeter.apache.org/usermanual/build-web-test-plan.html).
+2. Check in the JMX file and supporting files in a repository
+
+## AKS setup
+
+1. Create AKS cluster with the help of how to create a AKS cluster(https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough-portal)
+2. Provide access to a Service Principal Name which would be used to run the JMX file in the cluster.
+
+## Steps to onboard and execute the pipeline:
+
+1. Fork the test execution YAML pipeline from the repository: JMeterAKSLoadTest(https://github.com/microsoft/JMeterAKSLoadTest.git)
+2. Folder structure looks like below
+   ![Folder Structure](./Images/folder-structure.png)
+
+3. Inside the JMeterFiles folder add the JMX and supporting files there
+   ![JMeter Files](./Images/JMeter-files.png)
+
+4. Overview on the test execution pipeline variables which can be added by users before running the pipeline -
+
+- Tenant – tenant id
+- NameSpace - namespace
+- ServiceConnection - azure service connection
+- KeyVaultName - key vault name for fetching the secrets used in the pipeline
+- SecretNames - list of secrets which can be fetched from the key vault e.g. "AKSSPNClientSecret, PerfTestClientSecret"
+- AKSResourceGroup - resource groups for keeping AKS resources
+- AKSRegion1 - Respective region name e.g. westus2
+- AKSRegion2 - Respective region name e.g. cus
+- AKSClusterNameRegion1 - cluster name of the respective region
+- AKSClusterNameRegion2 - cluster name of the respective region
+- AKSSPNClientId – service principal id used for connecting to AKS clusters
+- AKSSPNClientSecret – client secret used for connecting to AKS clusters
+- PerfTestResourceId – Resource Id for the API Auth
+- PerfTestClientId – Client Id for the API Auth
+- CSVFileNames – list of supported file names for execution like “users.csv,ids.csv”
+  ![Pipeline variables](./Images/pipeline-variables.png)
+
+5. Overview on the test execution pipeline parameters which can be configured at every run while running the pipeline -
+
+- IsMultiRegionEnabled - allows user to optionally choose to run their workloads in more than one region
```

Comment:
Plz use pascal case for first word  for variable description like - Allows user to optionally choose to run their workloads in more than one region

Reply:
done